### PR TITLE
Note about 1click installations

### PR DESCRIPTION
**Pull request by fkaminski**

### CODE REVIEW

File: docs/manual/installation/system-requirements.de.md [hunk 1]
```diff
@@ -343,5 +343,11 @@ erfordern. Zum Glück sind sie nur die Ausnahme von der Regel. Die Provider-spez
 Contao-Hosting erhältst du bei den [Contao-Partnern](https://contao.org/de/contao-partner.html) in der 
 Leistungskategorie »Webhosting«.
 
+{{% notice "note" %}}
+Einige Service Provider bieten »1click« Installationen, auch für Contao, an. Für das beste Nutzungserlebnis empfehlen wir, bei der Installation
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Einige Hosting-Anbieter bieten 1-Klick-Installationen, auch für Contao, an. Für das beste Nutzungserlebnis empfehlen wir jedoch, bei der Installation 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

File: docs/manual/installation/system-requirements.de.md [hunk 2]
```diff
@@ -344,9 +344,8 @@ Contao-Hosting erhältst du bei den [Contao-Partnern](https://contao.org/de/cont
 Leistungskategorie »Webhosting«.
 
 {{% notice "note" %}}
-Einige Service Provider bieten »1click« Installationen, auch für Contao, an. Für das beste Nutzungserlebnis empfehlen wir, bei der Installation 
-von Contao, den Contao Manager oder die Konsole zu verwenden. Alle anderen Installationsmöglichkeiten, können zwar auch zum Ziel führen, 
-der Weg dorthin ist aber mitunter sehr steinig.
+Einige Hosting-Anbieter bieten 1click-Installationen, auch für Contao, an. Für das beste Nutzungserlebnis empfehlen wir jedoch, bei der
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Einige Hosting-Anbieter bieten 1-Klick-Installationen an. Für das beste Nutzungserlebnis empfehlen wir jedoch, bei der 
```

File: docs/manual/installation/quickstart.de.md [hunk 3]
```diff
@@ -7,8 +7,9 @@ aliases:
 weight: 15
 ---
 
-Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die [Long Term Support Version](https://docs.contao.org/manual/de/installation/contao-aktualisieren/#long-term-support-versionen) mit dem Contao Manager
-installieren willst. Das ist der einfachste und für Einsteiger empfohlene Weg.
+Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die [Long Term Support Version](https://docs.contao.org/manual/de/installation/contao-aktualisieren/#long-term-support-versionen) mit dem Contao Manager installieren willst. Das ist der einfachste und für Einsteiger 
+empfohlene Weg. Einige Hosting-Anbieter bieten 1click-Installationen, auch für Contao, an. Für das beste Nutzungserlebnis empfehlen wir jedoch,
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
empfohlene Weg. Einige Hosting-Anbieter bieten 1-Klick-Installationen an. Für das beste Nutzungserlebnis empfehlen wir jedoch, 
```

File: docs/manual/installation/system-requirements.en.md [hunk 4]
```diff
@@ -333,8 +333,8 @@ hosting packages for Contao from the [Contao partners](https://contao.org/en/con
 category "Web hosting".
 
 {{% notice "note" %}}
-Some service providers offer "1click" installations, also for Contao. For the best user experience, we recommend using the Contao Manager 
-or the console. All other installation options can also lead to the goal, but the way there is sometimes very rocky.
+Some hosting providers offer 1click installations, also for Contao. However, for the best user experience, we recommend using the
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.en.md]:
```suggestion
Some hosting providers offer 1-click installations. However, for the best user experience, we recommend using the 
```

File: docs/manual/installation/quickstart.en.md [hunk 5]
```diff
@@ -6,8 +6,9 @@ aliases:
 weight: 15
 ---
 
-We assume here that you want to install either the latest version or the [Long Term Support Version](https://docs.contao.org/manual/en/installation/update-contao/#long-term-support-versions) with the Contao Manager.
-This is the easiest and recommended way for beginners.
+We assume here that you want to install either the latest version or the [Long Term Support Version](https://docs.contao.org/manual/en/installation/update-contao/#long-term-support-versions) with the Contao Manager. This is the easiest and recommended way for beginners. Some hosting 
+providers offer 1click installations, also for Contao. However, for the best user experience, we recommend using the Contao Manager or the console.
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
providers offer 1-click installations. However, for the best user experience, we recommend using the Contao Manager or the console.
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]

**Comment by fkaminski** [comment]
Thank you Bjarke

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**